src/components/TextInput/BaseTextInput.js

```diff
@@ -30,6 +30,7 @@ class BaseTextInput extends Component {
             passwordHidden: props.secureTextEntry,
             textInputWidth: 0,
             prefixWidth: 0,
+            selection: props.selection,
```

Comment:
@eVoloshchak sorry I know it has been a long time, but what is the purpose of copying the `props.selection` to an internal state instead of just using it directly the props like `selection={this.props.selection}` here:

https://github.com/eVoloshchak/App/blob/d072ac100ae93cd4c6a1665777451caa72550853/src/components/TextInput/BaseTextInput.js#L277

Reply:
Check https://github.com/Expensify/App/pull/10147#discussion_r939578337 So apparently we used state to delay the selection by one cycle to fix a bug in iOS Safari.

We have this PR https://github.com/Expensify/App/pull/16825 that fixed a similar bug on Web. So maybe the original bug is fixed now and we can pass the prop directly.

Curious though, what are you doing :grin: @aldo-expensify

Reply:
> Check https://github.com/Expensify/App/pull/10147#discussion_r939578337 So apparently we used state to delay the selection by one cycle to fix a bug in iOS Safari.

Thanks for the explanation :). I think I would have been good to leave a comment so we don't just remove it in the future.

> Curious though, what are you doing 😁 @aldo-expensify

Just reviewing this PR: https://github.com/Expensify/App/pull/18579/files and I got curious about that.